plugins/inputs/varnish/varnish.go

```diff
@@ -32,6 +33,9 @@ var defaultStats = []string{"MAIN.cache_hit", "MAIN.cache_miss", "MAIN.uptime"}
 var defaultBinary = "/usr/bin/varnishstat"
 
 var sampleConfig = `
+  ## If running as a restricted user you can prepend sudo for additional access:
+	sudo = true
```

Comment:
Lets call it `use_sudo` to match the fail2ban plugin.  Indent the line only 2 spaces, set it to false (since that's default), and comment it out with a single hash (because it is not required).
```
  ## If running as a restricted user you can prepend sudo for additional access:
  # sudo = false
```

In the plugin README we should add a paragraph that discusses permissions and recommends adding the telegraf user to the varnish group over using sudo.  It should show how to give the telegraf user password-less sudo access to the varnishstat binary.